src/components/app.jsx

```diff
@@ -41,17 +51,16 @@ class App extends React.Component {
   saveCards() {
     let cards = this.state.cards;
     cards = JSON.stringify(cards);
-    localStorage.setItem('flash-cards', cards);
+    localStorage.setItem(['flash-cards', cards]);
   }
 
   addCards(card) {
-    this.setState({
-        cards: [...this.state.cards, card]
-    }, () => {this.saveCards()});
+    this.setState(prevState => ({
+        cards: [...prevState.cards, card]
+    }));
   }
 
   render() {
-    console.log("Cards From App ", this.state.cards)
```

Comment:
You should be using this `console.log` to ensure your `card` array is being set properly from localStorage when the App loads. But good job removing it for your pull request.